### PR TITLE
PLAT-1217 | Enable tests for Vue components

### PR DESCRIPTION
**Pull request by rogatty**
https://bethink.atlassian.net/browse/PLAT-1217

### CODE REVIEW

File: resources/assets/js/components/global/splashscreens/SplashScreen.spec.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+	}
+
+	const axiosGetFake = sinon.fake.returns({data: data});
+	sinon.replace(axios, 'get', axiosGetFake);
+}
+
+async function waitForLoadingState() {
+// We need to wait for `this.isLoading = true`, didn't find a better method yet
+	await localVue.nextTick();
+	await localVue.nextTick();
```
Comment (kvas-damian) [inline resources/assets/js/components/global/splashscreens/SplashScreen.spec.js]: I understand `nextTick` but why twice?

File: resources/assets/js/components/global/splashscreens/SplashScreen.spec.js [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+				currentUserSubscriptionStatus: () => 'expired',
+			},
+		});
+
+		fakeFetchOrders();
+
+		const component = getComponent({store});
+		await waitForLoadingState();
+
+		expect(component.html()).to.include('<wnl-splash-screen-subscription-expired');
```
Comment (kvas-damian) [inline resources/assets/js/components/global/splashscreens/SplashScreen.spec.js]: Why you sometimes expect stub and in other places not?

Comment (rogatty) [reply resources/assets/js/components/global/splashscreens/SplashScreen.spec.js]: A mistake, fixed in https://github.com/bethinkpl/wnl-platform/pull/1415/commits/e3eded80b2db8bd7c681001b2884f7c0697423ac

File: resources/assets/js/components/global/splashscreens/SplashScreen.spec.js (hunk 1 above)
Comment (rogatty) [reply resources/assets/js/components/global/splashscreens/SplashScreen.spec.js]: I've added explanation in https://github.com/bethinkpl/wnl-platform/pull/1415/commits/5d341bdcf24581c7e4dd0fc33249480a98f07a93

**Pull request merged**